app/src/main/java/nl/hogeschoolrotterdam/projectb/MemoryEditActivity.java

```diff
@@ -243,6 +253,145 @@ public void onClick(View v) {
+                break;
+            case 15:
+                foundIcon = R.drawable.ic_map_iron_throne;
+                break;
+            case 16:
+                foundIcon = R.drawable.ic_map_job;
+                break;
+            case 17:
+                foundIcon = R.drawable.ic_map_landscape;
+                break;
+            case 18:
+                foundIcon = R.drawable.ic_map_money;
+                break;
+            case 19:
+                foundIcon = R.drawable.ic_map_mood_extremely_happy;
+                break;
+            case 20:
+                foundIcon = R.drawable.ic_map_mood_extremely_sad;
+                break;
+            case 21:
+                foundIcon = R.drawable.ic_map_mood_happy;
+                break;
+            case 22:
+                foundIcon = R.drawable.ic_map_mood_sad;
+                break;
+            case 23:
+                foundIcon = R.drawable.ic_map_mood_very_happy;
+                break;
+            case 24:
+                foundIcon = R.drawable.ic_map_mood_very_sad;
+                break;
+            case 25:
+                foundIcon = R.drawable.ic_map_music;
+                break;
+            case 26:
+                foundIcon = R.drawable.ic_map_painting;
+                break;
+            case 27:
+                foundIcon = R.drawable.ic_map_party;
+                break;
+            case 28:
+                foundIcon = R.drawable.ic_map_pets;
+                break;
+            case 29:
+                foundIcon = R.drawable.ic_map_pool;
+                break;
+            case 30:
+                foundIcon = R.drawable.ic_map_restaurant;
+                break;
+            case 31:
+                foundIcon = R.drawable.ic_map_school;
+                break;
+            case 32:
+                foundIcon = R.drawable.ic_map_sexy;
+                break;
+            case 33:
+                foundIcon = R.drawable.ic_map_shield;
+                break;
+            case 34:
+                foundIcon = R.drawable.ic_map_spa;
+                break;
+            case 35:
+                foundIcon = R.drawable.ic_map_speakers;
+                break;
+            case 36:
+                foundIcon = R.drawable.ic_map_starred;
+                break;
+            case 37:
+                foundIcon = R.drawable.ic_map_store;
+                break;
+            case 38:
+                foundIcon = R.drawable.ic_map_styles;
+                break;
+            case 39:
+                foundIcon = R.drawable.ic_map_teathers;
+                break;
+            case 40:
+                foundIcon = R.drawable.ic_map_tourist;
+                break;
+            case 41:
+                foundIcon = R.drawable.ic_map_want_to_go;
+                break;
+            case 42:
+                foundIcon = R.drawable.ic_map_world;
+                break;
+            default:
+                foundIcon = R.drawable.ic_map_adefaultl;
```

Comment:
Are you 100% sure that all these icons are on the right position?

Reply:
yes!